### PR TITLE
Chore/Remove old histogram

### PR DESCRIPTION
**Pull request by Tanguylo**

### CODE REVIEW

File: plot_data/core.py [hunk 1]
```diff
@@ -882,7 +882,19 @@ def __init__(self, elements: List[Sample] = None, axes: List[str] = None, point_
...
         self.surface_style = surface_style
-        super().__init__(width=width, height=height, type_="scattermatrix", name=name)
+        self.plots = self._build_multiplot()
+        self.initial_view_on = True
+        super().__init__(width=width, height=height, type_="multiplot", name=name)
+
+    def _build_multiplot(self):
+        subplots = []
+        for line in self.elements[0].values:
+            for col in self.elements[0].values:
```
Comment (GhislainJ) [inline plot_data/core.py:892]: Did you try assigning subplots list in a one-liner comprehension list to see if it is elegant ? Could probably be : 

```py
element = self.elements[0] # Name might not be good
return [Histogram(col) for row in element.values for col in element.values
        if row == col else Scatter(row, col)]
```

Now that I think about it, isn't it weird that we loop twice for col and rows on self.elements[0].values ? Is it what is intended ?

File: src/multiplots.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import {PlotData, Interactions} from './plot-data';
 import {Point2D} from './primitives';
 import { Attribute, PointFamily, check_package_version, Window, TypeOf, equals, Sort, export_to_txt, RubberBand } from './utils';
-import { PlotContour, PlotScatter, ParallelPlot, PrimitiveGroupContainer, newHistogram, Histogram, Frame } from './subplots';
+import { PlotContour, PlotScatter, ParallelPlot, PrimitiveGroupContainer, newHistogram, Frame } from './subplots';
```
[the remark this answers is not in the document]
Comment (Tanguylo) [reply src/multiplots.ts:4]: Just forgot it, thanks

File: plot_data/core.py [hunk 3]
```diff
@@ -882,7 +882,19 @@ def __init__(self, elements: List[Sample] = None, axes: List[str] = None, point_
...
         self.point_style = point_style
         self.surface_style = surface_style
-        super().__init__(width=width, height=height, type_="scattermatrix", name=name)
+        self.plots = self._build_multiplot()
+        self.initial_view_on = True
+        super().__init__(width=width, height=height, type_="multiplot", name=name)
+
+    def _build_multiplot(self):
+        subplots = []
+        for line in self.elements[0].values:
```
[the remark this answers is not in the document]
Comment (Tanguylo) [reply plot_data/core.py:891]: Done

File: plot_data/core.py (hunk 1 above)
Comment (Tanguylo) [reply plot_data/core.py]: I did not know it was possible to double loop in one line, thanks !

**Review by GhislainJ — APPROVED** [review]

**Pull request merged**